src/test/java/cd/go/contrib/elasticagent/executors/GetPluginConfigurationExecutorTest.java

```diff
@@ -46,48 +48,60 @@ public void assertJsonStructure() throws Exception {
-                "    \"display-name\": \"Go Server URL\",\n" +
+                "    \"display-name\": \"GoCD server URL\",\n" +
                 "    \"required\": false,\n" +
                 "    \"secure\": false,\n" +
                 "    \"display-order\": \"0\"\n" +
                 "  },\n" +
                 "  \"auto_register_timeout\": {\n" +
-                "    \"display-name\": \"Agent auto-register Timeout (in minutes)\",\n" +
+                "    \"display-name\": \"Agent auto-register timeout (in minutes)\",\n" +
                 "    \"default-value\": \"10\",\n" +
-                "    \"required\": true,\n" +
+                "    \"required\": false,\n" +
                 "    \"secure\": false,\n" +
                 "    \"display-order\": \"1\"\n" +
                 "  },\n" +
                 "  \"pending_pods_count\": {\n" +
-                "    \"display-name\": \"Maximum Pending Kuberneted Pods Count\",\n" +
+                "    \"display-name\": \"Maximum pending pods\",\n" +
                 "    \"default-value\": \"10\",\n" +
-                "    \"required\": true,\n" +
+                "    \"required\": false,\n" +
                 "    \"secure\": false,\n" +
                 "    \"display-order\": \"2\"\n" +
                 "  },\n" +
                 "  \"kubernetes_cluster_url\": {\n" +
-                "    \"display-name\": \"Kubernetes Cluster URL\",\n" +
+                "    \"display-name\": \"Cluster URL\",\n" +
+                "    \"required\": false,\n" +
+                "    \"secure\": false,\n" +
+                "    \"display-order\": \"5\"\n" +
+                "  },\n" +
+                "  \"kubernetes_cluster_ca_cert\": {\n" +
+                "    \"display-name\": \"Cluster ca certificate\",\n" +
+                "    \"required\": false,\n" +
+                "    \"secure\": true,\n" +
+                "    \"display-order\": \"6\"\n" +
+                "  },\n" +
+                "  \"authentication_strategy\": {\n" +
+                "    \"display-name\": \"Authentication strategy\",\n" +
                 "    \"required\": true,\n" +
                 "    \"secure\": false,\n" +
                 "    \"display-order\": \"3\"\n" +
                 "  },\n" +
-                "  \"kubernetes_cluster_username\": {\n" +
-                "    \"display-name\": \"Kubernetes Cluster Username\",\n" +
+                "  \"oauth_token\": {\n" +
+                "    \"display-name\": \"Oauth Token\",\n" +
                 "    \"required\": false,\n" +
-                "    \"secure\": false,\n" +
+                "    \"secure\": true,\n" +
                 "    \"display-order\": \"4\"\n" +
                 "  },\n" +
-                "  \"kubernetes_cluster_password\": {\n" +
-                "    \"display-name\": \"Kubernetes Cluster Password\",\n" +
+                "  \"client_key_data\": {\n" +
+                "    \"display-name\": \"Client key data\",\n" +
```

Comment:
Client key data => Client key

Reply:
Following k8s standard, in `~/kube/config` where `certificate-authority` is path to `.crt` file and `certificate-authority-data` is content of `.crt` file.